examples/twilio_api_v2010.yaml

```diff
@@ -1,5 +1,30 @@
 components:
   schemas:
+    test_enum_permission:
+      enum:
+        - get-all
+        - post-all
+      type: string
+    test_enum_status:
+      enum:
+        - in-progress
+        - paused
+        - stopped
+        - processing
+        - completed
+        - absent
+      type: string
+    recording_enum_source:
+      enum:
+        - DialVerb
+        - Trunking
+      type: string
+    payments_enum_bank_account_type:
+      enum:
+        - consumer-checking
+        - consumer-savings
+        - commercial-checking
+      type: string
```

Comment:
I say just reduce these down to 1 enum to make the fixture more concise.

Reply:
@childish-sambino Test are passing for go-lang. We did raise PR for java. As those changes in different commit. If you approve it then we can raise another PR for java

Reply:
We will remove extra enums before merging to main

Reply:
Prefer to not make extra changes after getting approval. Think it's best to trim them down first, then review, then merge for a smooth process.